### DISCUSSION TITLE
Translation for Horizon and Telescope projects

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#17] [Q&A]
@Laravel-Lang/laravel-lang, yesterday I implemented the ability to translate the interface of the Laravel [`Horizon`](https://github.com/laravel/horizon/pull/1175) and [`Telescope`](https://github.com/laravel/telescope/pull/1244) projects, but today Taylor denied us this opportunity, closing the PR without explaining the reasons.

As practice shows, Taylor pays attention to some features when developers are interested in them. What would you say if I ask you to be active in closed PR?

What do you think, we need it?

- https://github.com/laravel/horizon/pull/1175
- https://github.com/laravel/telescope/pull/1244
- https://github.com/laravel/framework/discussions/43699

I have prepared the changes on our side, but I cannot accept them until this functionality is added to the parent projects, because without it the changes will not work.

- https://github.com/Laravel-Lang/lang/pull/2316
- https://github.com/Laravel-Lang/lang/pull/2318

**Comment by makowskid** [comment]
yup! definitely

**Reply by mohamedsabil83** [reply]
Agree. we sure do
